elastic-search-support/src/main/resources/casesIndexSettings.json

```diff
@@ -1,21 +1,28 @@
 {
-  "analysis":{
-    "analyzer":{
-      "phone_number_analyzer":{
-        "type":"custom",
-        "tokenizer":"keyword",
-        "filter":[
-          "whitespace_remove"
-        ]
+  "analysis": {
+      "analyzer": {
+          "phone_number_analyzer": {
+              "type": "custom",
+              "tokenizer": "keyword",
+              "filter": [
+                  "custom_shingle",
+                  "whitespace_remove"
+              ]
+          }
+      },
+      "filter": {
+          "custom_shingle": {
```

Comment:
Maybe we can come up with a different name for this, something which is more descriptive of what it actually does?